iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/Replace3TwoWindingsTransformersByThreeWindingsTransformersTest.java

```diff
@@ -0,0 +1,440 @@
+        return t3w.getAliasFromType("CGMES.TransformerEnd" + leg).orElseThrow().equals("transformerEnd-" + t2w.getId())
+                && t3w.getAliasFromType("CGMES.Terminal" + leg).orElseThrow().equals("terminal-" + t2w.getId())
+                && t3w.getAliasFromType("CGMES.RatioTapChanger" + leg).orElseThrow().equals("ratioTapChanger-" + t2w.getId())
+                && t3w.getAliasFromType("CGMES.PhaseTapChanger" + leg).orElseThrow().equals("phaseTapChanger-" + t2w.getId());
+    }
+
+    @Test
+    void testReportNode() throws IOException {
+        network.getTwoWindingsTransformer(t2w1.getId()).setProperty("t2w1 property1", "t2w1-value1");
+        network.getTwoWindingsTransformer(t2w2.getId()).setProperty("t2w2 property1", "t2w2-value1");
+        network.getTwoWindingsTransformer(t2w2.getId()).setProperty("t2w2 property2", "t2w2-value2");
+        network.getTwoWindingsTransformer(t2w3.getId()).setProperty("t2w3 property1", "t2w3-value1");
+        network.getTwoWindingsTransformer(t2w1.getId()).addAlias("t2w1-alias1");
+        network.getTwoWindingsTransformer(t2w1.getId()).addAlias("t2w1-alias2");
+        network.getTwoWindingsTransformer(t2w2.getId()).addAlias("t2w2-alias1");
+        network.getTwoWindingsTransformer(t2w3.getId()).addAlias("t2w3-alias1");
+        ReportNode reportNode = ReportNode.newRootReportNode().withMessageTemplate("test", "test reportNode").build();
+        Replace3TwoWindingsTransformersByThreeWindingsTransformers replace = new Replace3TwoWindingsTransformersByThreeWindingsTransformers();
+        replace.apply(network, reportNode);
+
+        StringWriter sw1 = new StringWriter();
+        reportNode.print(sw1);
+        assertEquals("""
+                + test reportNode
+                   + Replaced 3 TwoWindingsTransformers by ThreeWindingsTransformer
+                      TwoWindingsTransformer 3WT-Leg1 removed
+                      TwoWindingsTransformer 3WT-Leg2 removed
+                      TwoWindingsTransformer 3WT-Leg3 removed
+                      Voltage level 3WT-Star-VL, its equipments and the branches it is connected to have been removed
+                      Property [t2w1 property1] of twoWindingsTransformer 3WT-Leg1 will be lost
+                      Property [t2w2 property1,t2w2 property2] of twoWindingsTransformer 3WT-Leg2 will be lost
+                      Property [t2w3 property1] of twoWindingsTransformer 3WT-Leg3 will be lost
+                      Alias [t2w1-alias2,t2w1-alias1] of twoWindingsTransformer 3WT-Leg1 will be lost
+                      Alias [t2w2-alias1] of twoWindingsTransformer 3WT-Leg2 will be lost
+                      Alias [t2w3-alias1] of twoWindingsTransformer 3WT-Leg3 will be lost
+                      ThreeWindingsTransformer 3WT-Leg1-3WT-Leg2-3WT-Leg3 created
+                """, TestUtil.normalizeLineSeparator(sw1.toString()));
+    }
+
+    @Test
+    void testReportNodeExtensions() throws IOException {
```

Comment:
Consider for more consistency to name the method ```reportNodeExtensionsTest``` instead

Reply:
Done